src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -112,6 +112,7 @@ function FilterBarUI(props: Props) {
         size="xs"
         onClick={() => setIsAddFilterPopoverOpen(true)}
         data-test-subj="addFilter"
+        aria-label="addFilter"
```

Comment:
Should our labels be more detailed? https://oui.opensearch.org/1.6/#/guidelines/accessibility

should @opensearch-project/opensearch-ux have some weigh in here what they should be?

Reply:
The purpose of `aria-label`s is mean to be human readable. If this is an id to the element that contains the element, should be using [`aria-labelledby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)

Reply:
@kavilla Can you add an image/ screenshot of the UI element the aria label is needed for? cc @abbyhu2000

Reply:
@virajsanghvi I think the purpose of this is to make the button aligned with accessibility requirements, so we wish to add `aria-label` so it is human readable.

Reply:
@lauralexis here is the screenshot for the button:
<img width="497" alt="Screenshot 2024-06-06 at 11 19 10 AM" src="https://github.com/opensearch-project/OpenSearch-Dashboards/assets/43937633/09ae80b3-14c0-4b8e-a9a0-52cc23f67970">

Reply:
@lauralexis the text is not used as the button label, it is just for accessibility requirement. I will change the aria label to `Add filter` as recommended.